modules/nf-core/lofreq/alnqual/main.nf

```diff
@@ -0,0 +1,51 @@
+process LOFREQ_ALNQUAL {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/lofreq:2.1.5--py310h47ef89e_10' :
+        'biocontainers/lofreq:2.1.5--py310h47ef89e_10' }"
+
+    input:
+    tuple val(meta), path(bam)
+    path(fasta)
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    lofreq \\
+        alnqual \\
+        $args \\
+        -b \\
```

Comment:
Is this a mandatory argument?

Reply:
It sets output as a bam file rather than sam